config/amazon_requests_proxies.json

```diff
@@ -0,0 +1,7 @@
+{
+      "enable": "n",
+      "user": "user",
+      "pass": "pass",
+      "ip_port": [{"http":"XXX.XXX.XXX.XXX:XXXX", "https": "XXX.XXX.XXX.XXX:XXXX"},

```

Comment:
Can you properly format this?

Reply:
Yeah. Can I use black for that? This is the first time I have made a json file